src/sync/synchronizer.rs

```diff
@@ -872,7 +872,7 @@ fn apply_inserts_to_localmap(
             let in_mem_inner_map = table_synchronizer
                 .in_memory_map
                 .entry(update.outer_key.clone())
-                .or_insert_with(HashMap::new);
+                .or_default();
```

Comment:
Updated this line as per the suggestion provided in the failed build log.
![image](https://github.com/pravega/pravega-client-rust/assets/122247323/879e2429-fb66-4fce-93a8-4ef50cbdd0ab)